### DISCUSSION TITLE
I am trying to add my FB birthday fundraiser onto my website through html or css

### DISCUSSION THREAD

**Discussion by ijustloveflexin** [community/community#147725] [Programming Help]
### Select Topic Area

Question

### Body

I just tried to post a question via answers.microsoft.com I'm not sure if it went through I'm trying to add a birthday fundraiser onto my website for the rest of December. This was the link for my question but it says thread not found. 

https://answers.microsoft.com/en-us/windows/forum/windows_10-networking/i-am-trying-to-add-my-fb-birthday-fundraiser-onto/774d548a-1781-43c9-8012-50f3b1774dff

[_redacted by moderator_]

**Comment by ijustloveflexin** [comment]
Also gmail closed my account for [_redacted by moderator_] this email is connected to my youtube, my google home speaker, among other things and I noticed that it was listed on my profile so I took it off because I can't receive email at that address anymore. It was under aliases though so I tried to add it back thinking I might be able to get email that was sent to that gmail account but it wouldn't let me add it back and i don't even know for sure if that would work.

**Comment by ebndev** [comment]
Hi @ijustloveflexin, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

I've removed your private information from this discussion as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by Ayesha0709** [comment]
**It would help if you could clarify exactly what you're trying to achieve.

If your goal is to embed your Facebook birthday fundraiser on your website, HTML and CSS alone won't be enough. HTML and CSS only control the structure and appearance of a page—they can't embed dynamic Facebook content by themselves.

If Facebook provides an official embed option or widget for fundraisers, that's the recommended approach. Otherwise, you can simply add a link or button that redirects visitors to your fundraiser page.

Could you also share:


- the URL of the fundraiser (if you're comfortable),
- whether you're trying to embed the full fundraiser or just link to it,
- and what you've already tried?**
